apps/sms/test/marionette/lib/views/conversation/accessors.js

```diff
@@ -122,9 +122,15 @@ ConversationAccessor.prototype = {
 
   waitToAppear: function() {
     var conversationPanel = this.client.helper.waitForElement(SELECTORS.main);
-    this.client.waitFor(function() {
-      return conversationPanel.location().x === 0;
+
+    this.client.waitFor(() => {
+      return conversationPanel.location().x === 0 && this.client.executeScript(
```

Comment:
Likely yes, kind of ```waitForViewToAppear(viewElement)``` somewhere in common place. Please, tell me if you want me to do this in this pr